unlock-js/src/walletService.js

```diff
@@ -279,6 +283,16 @@ export default class WalletService extends UnlockService {
     return version.purchaseKey.bind(this)(params, callback)
   }
 
+  /**
+   * Grants a key to an address
+   * @param {function} callback : callback invoked with the transaction hash
+   */
+  async grantKey(params = {}, callback) {
```

Comment:
It might be useful to note what params might consist of in the method documentation. It looks lockAddress is all that is required up top, but since its is delegating the call else where callers wont know unless they review the code

Reply:
That's a good point. We need to be better at this. versions actually require/enable different params. for example in v7, one can pass the lockManager, but other versions do not have that concept.

I think there will need to have a broard effort around docs.